Documentation/api-reference.md

```diff
@@ -2,22 +2,27 @@
 
 This document lists the APIs provided by AEP Roku SDK, along with code samples for API usage.
 
-
+- Core APIs
+  - [AdobeAEPSDKInit](#AdobeAEPSDKInit)
+  - [getVersion](#getVersion)
+  - [setLogLevel](#setLogLevel)
+  - [shutdown](#shutdown)
+  - [updateConfiguration](#updateConfiguration)
+- Consent APIs
+  - [setConsent](#setConsent)
 - Edge APIs
-    - [AdobeAEPSDKInit](#AdobeAEPSDKInit)
-    - [getExperienceCloudId](#getExperienceCloudId)
-    - [getVersion](#getVersion)
-    - [resetIdentities](#resetIdentities)
-    - [sendEvent](#sendEvent)
-    - [(optional) setExperienceCloudId](#setExperienceCloudId)
-    - [setLogLevel](#setLogLevel)
-    - [shutdown](#shutdown)
-    - [updateConfiguration](#updateConfiguration)
+  - [sendEvent](#sendEvent)
+- Identitiy APIs
+  - [getExperienceCloudId](#getExperienceCloudId)
+  - [resetIdentities](#resetIdentities)
+  - [(optional) setExperienceCloudId](#setExperienceCloudId)
 - Media APIs
-    - [createMediaSession](#createMediaSession)
-    - [sendMediaEvent](#sendMediaEvent)
+  - [createMediaSession](#createMediaSession)
+  - [sendMediaEvent](#sendMediaEvent)
+- Util APIs
+  - [_adb_ISO8601_timestamp](#_adb_ISO8601_timestamp)
```

Comment:
Can you remove the leading underscore and rename this method to `adb_ISO8601_timestamp`?

Reply:
Sure, I wanted to also get confirmation that we are okay to make this public that's why did not update it. 
Also how do you think about adding it a ADB_Helper class. 

```
function AEPSdkHelper

  function getISO8601Timestamp
     return _adb_ISO8601_timestamp();
  end function
  
end function 
```

Usage:
```
aepSdkHelper = AEPSdkHelper()
ts = aepSdkHelper. getISO8601Timestamp()
```

Reply:
Why do we need to expose this API?

Reply:
@stcheng-adobe there is no straight forward way to create ISO timestamp on Roku and the setConsent API will error if customer passes Timestamp in different format. So this helper will make it easy for them to get the ts in right format.

Reply:
I reviewed the PR again, and the term ‘timestamp’ isn’t very clear. I prefer changing the helper’s name to [AEPSDKUtils.getISO8601UTCDateWithMilliseconds](https://github.com/adobe/aepsdk-core-android/blob/main/code/core/src/main/java/com/adobe/marketing/mobile/util/TimeUtils.kt#L90C9-L90C42) to align with our mobile implementation?

Reply:
@praveek sounds good!

Reply:
@praveek @stcheng-adobe 
I was searching the Roku docs again for this and I do see ISO dateTime method. And We can just mention that method to be used for this and not document our internal method.  

```
  dateTime = CreateObject("roDateTime")
  
  isoDate = dateTime.ToISOString("milliseconds")
  isoDateFromSDK = _adb_ISO8601_timestamp()
  
  print("roku method iso date:" + FormatJson(isoDate))
  print("sdk method iso date:" + FormatJson(isoDateFromSDK))
```

Output:
```
roku method iso date:"2024-08-30T03:11:12.395Z"
sdk method iso date:"2024-08-30T03:11:12.395Z"
```

We can also update our method to use this for our logic as well. But I will have to understand why we did not use this in the first place.